README.md

```
@@ -1,337 +1,67 @@
-# Kode Start by Kobe
+# Desafio Kobe Apps - Aplicativo Rick and Morty API 
 
-Neste repositório se encontram os aplicativos desenvolvidos durante as aulas, outros projetos desenvolvidos ao longo do programa e, também, documentações
+Este repositório contém a solução desenvolvida como parte obrigatória na participação do 3º Kode Start da Kobe Apps.
 
-## Links úteis
-- Dart Tour: https://dart.dev/guides/language/language-tour
-- Projects: https://github.com/kobeapps/kode-start/tree/main/projects
+O objetivo foi construir um aplicativo em Flutter para listar e exibir detalhes dos personagens do desenhor Rick and Morty através da API de Rick and Morty (https://rickandmortyapi.com/).
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)